ui/app/AppLayouts/Profile/Sections/Ens/Welcome.qml

```diff
@@ -0,0 +1,275 @@
+                anchors.leftMargin: 24
+                anchors.top: element3Subtitle.bottom
+                anchors.topMargin: 24
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element4Title
+                text: qsTr("10 SNT to register")
+                anchors.left: element4Number.right
+                anchors.leftMargin: 24
+                anchors.top: element3Subtitle.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element4Subtitle
+                text: qsTr("Register once to keep the name forever. After 1 year you can release the name and get your SNT back.")
+                anchors.left: element4Number.right
+                anchors.leftMargin: 24
+                anchors.top: element4Title.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.pixelSize: 14
+            }
+
+
+            StyledText {
+                id: element5Number
+                text: "@"
+                anchors.left: parent.left
+                anchors.leftMargin: 24
+                anchors.top: element4Subtitle.bottom
+                anchors.topMargin: 24
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element5Title
+                text: qsTr("Already own a username?")
+                anchors.left: element5Number.right
+                anchors.leftMargin: 24
+                anchors.top: element4Subtitle.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.weight: Font.Bold
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: element5Subtitle
+                text: qsTr("You can verify and add any usernames you own in the next steps.")
+                anchors.left: element5Number.right
+                anchors.leftMargin: 24
+                anchors.top: element5Title.bottom
+                anchors.topMargin: 24
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.pixelSize: 14
+            }
+
+            StyledText {
+                id: poweredBy
+                text: qsTr("Powered by Ethereum Name Services")
+                anchors.left: element5Number.right
+                anchors.leftMargin: 24
+                anchors.top: element5Subtitle.bottom
+                anchors.topMargin: 40
+                anchors.right: parent.right
+                wrapMode: Text.WordWrap
+                font.pixelSize: 11
+            }
```

Comment:
All those Texts seem to be stacked on top of each other with almost always the same spacing. You don't have to change it, but I'm just letting you know that in a case like this, `Row` is pretty nice, because it will place each Text one on top of each other with the wanted spacing (24 in your case)